docker/ccf_keys/check-for-ccf-keys.sh

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+sleep 3
```

Comment:
this really adds nothing. there is already a sleep below.

Reply:
thank you, yes, this is unintentional, spill over from local testing. will remove it.